test/RelayHubGasCalculations.test.ts

```diff
@@ -39,14 +46,14 @@ contract('RelayHub gas calculations', async function ([_, relayOwner, relayAddre
     post: 2080
   }
 
-  let relayHub
-  let recipient
-  let paymaster
+  let relayHub: RelayHubInstance
```

Comment:
something is strange. `relayHub` is an instance, of type `RelayHub`. but what's the type `RelayHubInstance` ?

Reply:
Generated type of `RelayHub` truffle contract instance.
`relayHub` is an instance of `RelayHubInstance`.
RelayHub is an instance of artifact `TruffleContract<RelayHub>`